letsencrypt_s3front/installer.py

```diff
@@ -66,7 +69,9 @@ def deploy_cert(self, domain, cert_path, key_path, chain_path, fullchain_path):
         cert_id = response['ServerCertificateMetadata']['ServerCertificateId']
         # Update CloudFront config to use the new one
         cf_cfg = cf_client.get_distribution_config(Id=self.conf('cf-distribution-id'))
-        cf_cfg['DistributionConfig']['ViewerCertificate']['IAMCertificateId'] = cert_id
+        cf_cfg['DistributionConfig']['ViewerCertificate']['IAMCertificateId'] = cert_id        
+        cf_cfg['DistributionConfig']['ViewerCertificate']['Certificate'] = cert_id
```

Comment:
This fails on the AWS API. There is no `Certificate` or  `CertificateSource`, Where were you going with it?